custom_components/plugwise/const.py

```diff
@@ -44,16 +44,24 @@
 DEV_CLASS: Final = "dev_class"
 NONE : Final = "None"
 MODE: Final = "mode"
-SETPOINT: Final = "setpoint"
-SETPOINT_HIGH: Final = "setpoint_high"
-SETPOINT_LOW: Final = "setpoint_low"
+TARGET_TEMP: Final = "setpoint"
```

Comment:
see previous comment

Reply:
As a follow-up to the above comment: that's the nice thing of a Constant, they don't have to be the same :)